outlook/src/taskpane/components/App.tsx

```diff
@@ -31,10 +32,12 @@ export interface AppState {
     connect: (token) => void,
     disconnect: () => void,
     getConnectionToken: () => void,
+    getUserCompaniesString: () => string,
     isConnected: () => Boolean,
     cancelRequests: () => void,
     addRequestCanceller: (canceller: () => void) => void,
     showTopBarMessage: (enrichmentInfo?: EnrichmentInfo) => void,
+    setUserCompanies: (userCompanies: number[]) => void,
```

Comment:
Just being that guy: setUserCompanies should be defined before showTopBarMessage ^^

Reply:
Done :)